gradle/publish.gradle

```diff
@@ -7,3 +11,20 @@ publishing {
         }
     }
 }
+
+bintray {
+    user = System.env.BINTRAY_USERNAME
+    key = System.env.BINTRAY_PASSWORD
+    publish = true
+    pkg {
+        repo = 'releases'
+        name = 'http-remoting'
+        userOrg = 'palantir'
+        licenses = ['Apache-2.0']
+        publications = ['nebula','nebulaShadow']
```

Comment:
this is still a bit weird since not all projects have a `nebulaShadow` publication but it doesn't seem to hurt (there is a warning about the publication not being found)